ps_themecusto.php

```diff
@@ -76,7 +76,7 @@ public function __construct()
         $this->css_path = $this->_path.'views/css/';
         $this->img_path = $this->_path.'views/img/';
         $this->logo_path = $this->_path.'logo.png';
-        $this->module_path = $this->_path;
+        $this->module_path = dirname(__FILE__);
```

Comment:
@Adrienaddons why not use `$this->_path` isn't it safer than `dirname(__FILE__)`? cc @Quetzacoalt91

Reply:
I can already say the the use of `__DIR__` and `__FILE__` can bring some issues in some case, in particular when you use symbolic links. and try to reach the parent folder

I currently have it with the 1-click upgrade module.

Reply:
Exactly, we have `$this->_path`, `$this->local_path`, IMO there's no point of not using them